CHANGELOG.adoc

```diff
@@ -36,6 +36,7 @@ When end-users click on _New Object_ on a semantic element, and select a `ViewUs
 - https://github.com/eclipse-syson/syson/issues/1210[#1210] general-view] Add the `parameters` compartment on `ActionDefinition` in the _General View_ diagram.
 - https://github.com/eclipse-syson/syson/issues/1267[#1267] [general-view] Improve the New Perform tool selection dialog to display existing actions in a tree.
 - https://github.com/eclipse-syson/syson/issues/1257[#1257] [general-view] Add `perform actions` compartment in Parts
+- https://github.com/eclipse-syson/syson/issues/1277[#1277] [general-view] Add `perform actions` compartment in Actions
```

Comment:
Add the `perform actions` compartment on `Action` in the _General View_ diagram.